cluster/up.sh

```diff
@@ -25,11 +25,23 @@ if [[ "$KUBEVIRT_PROVIDER" != external ]]; then
     $(kubevirtci::path)/cluster-up/up.sh
 fi
 
-# Deploy CNA
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/namespace.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/network-addons-config.crd.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/operator.yaml
-./cluster/kubectl.sh create -f ./hack/cna/cna-cr.yaml
+if [[ "$CNAO_PROVIDER" != external ]]; then
+  # Deploy CNAO
+  ./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/namespace.yaml
+  ./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/network-addons-config.crd.yaml
+  ./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/operator.yaml
+fi
+
+# Deploy NetworkAddonsConfig
+if [[ -z $(./cluster/kubectl.sh get pod -A -l app=kubemacpool) ]]; then
+  cna=$(./cluster/kubectl.sh get networkaddonsconfig -o custom-columns=:.metadata.name --no-headers)
+  if [[ -z $cna ]]; then
+    ./cluster/kubectl.sh apply -f ./hack/cna/cna-cr.yaml
+  fi
+
+  # Deploy kubeMacPool with CNAO
+  ./cluster/kubectl.sh patch networkaddonsconfig cluster --type json -p='[{"op":"replace","path":"/spec/kubeMacPool","value":{}}]'
```

Comment:
why do we do this? isn't it always `{}` ?

```
  spec:
    kubeMacPool: {}
    linuxBridge: {}
    multus: {}
    nmstate: {}
    ovs: {}

```

Reply:
This patch adds the line `kubeMacPool: {}`